Library/ViewModels/ManagePledgeViewModel.swift

```diff
@@ -468,12 +469,9 @@ private func managePledgePaymentMethodViewData(
 
 private func managePledgeSummaryViewData(
   with project: Project,
+  backedReward: Reward,
```

Comment:
We're now passing in the backed reward instead of trying to retrieve it from the `Project`'s `personalization` property as the latter is unavailable in the creator context.